Makefile

```diff
@@ -0,0 +1,55 @@
+DEPDIR=$(PWD)/deps
+BUILDDIR=$(PWD)/build
+
+LIBERASURE=$(DEPDIR)/liberasurecode
+ISALSRC=$(DEPDIR)/isa-l
+DEV_1_0 := $(shell git show origin/development/1.0 | grep commit | cut -d" " -f 2)
```

Comment:
Why do we replace the hardcoded `983e1d15` by this dynamic resolution?

Seems like it's used for the linter because we don't want to lint upstream code, so I don't think we should keep updating it (or we will never lint our own code).

Or I misunderstood something?

Reply:
I understood that we lint the code that has changed from the current branch ("bugfix/..., feature/....") to development/1.0 .

If we merge upstream code we still need a PR to do so, and it should follow the same rule regarding code formatting. 

Here my point is just to make a manual operation become automatic.

@benoit-a why do we specify the commit sha-1 in the linter ?

Reply:
Please don't replace it, the use of the revision is to prevent golangci-lint from complaining for stuff written BEFORE we activated it.
In a nutshell, the linter will only complain on new issue since we first activated. The sha1 is there on purpose.

If it bothers you, it would be acceptable to fix all the linter violations (in a different PR) and get rid of it.

Reply:
it does not bothers me, I can keep it. 

If I understand, the sha1 will always stay the same ? regardless of our updates or any rebase with the upstream repository we will always keep this sha1 ? if this is the case I would use a variable with a name that can explain that and put the variable instead of the pure sha1 value.

Reply:
I kept the value hard-coded. the linter only start from that specific commit.